orchestration/SetupWorkDirs.py

```diff
@@ -73,15 +72,15 @@ def run(self):
                 jobTemplate = Template(templateFile.read())
 
             params = {
+                'jobWorkingDir': baseWorkDir,
                 'input': str(Path(basket['basketPath']).parent),
                 'state': stateDir,
                 'static': self.staticLocation,
                 'working': workingDir,
                 'output': self.outputLocation,
                 'containerPath': self.containerPath,
                 'inputFolder': pairName,
-                'outputSRS': self.outputSRS,
-                'finalJobTarget': 'CleanupCompletedProductInputs' if self.cleanupBasket else 'ConvertToTif'
+                'outputSRS': self.outputSRS
```

Comment:
Removing finalJobTarget and cleanupBasket param to try and decouple things - can change template if needed.